src/components/hand-controls.js

```diff
@@ -185,7 +193,7 @@ module.exports.Component = registerComponent('hand-controls', {
 
     // Set model.
     if (hand !== previousHand) {
-      this.loader.load(MODEL_URLS[hand], function (gltf) {
+      this.loader.load(MODEL_URLS[handModelStyle + hand.charAt(0).toUpperCase() + hand.slice(1)], function (gltf) {
```

Comment:
We can move this to a variable above to clean up a bit.
`var handmodel = MODEL_URLS[handModelStyle + hand.charAt(0).toUpperCase() + hand.slice(1);`